core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java

```diff
@@ -361,7 +361,7 @@ TransactionResponse fromTxn(Sep24Transaction txn, String lang)
   JwtToken buildRedirectJwtToken(String fullRequestUrl, JwtToken token, Sep24Transaction txn) {
     return JwtToken.of(
         fullRequestUrl,
-        token.getAccount(),
+        token.getSub(),
```

Comment:
Both `getSub()` and `getAccount()` return the same result, right? What does "sub" mean? I think "account" might be clearer.

Reply:
`sub` is the common `subject` field of a JWT. `account` is a fields of a SEP-10 JWT.